CRM/Core/Lock.php

```diff
@@ -244,11 +244,11 @@ public function isAcquired() {
    */
   public function hackyHandleBrokenCode($jobLog) {
     if (stristr($this->_name, 'job')) {
-      CRM_Core_Error::debug_log_message('lock acquisition for ' . $this->_name . '(' . $this->_id . ')' . ' attempted when ' . $jobLog . ' is not released');
+      \Civi::log()->debug('lock acquisition for ' . $this->_name . '(' . $this->_id . ')' . ' attempted when ' . $jobLog . ' is not released');
       throw new CRM_Core_Exception('lock acquisition for ' . $this->_name . '(' . $this->_id . ')' . ' attempted when ' . $jobLog . ' is not released');
```

Comment:
I put merge-on-pass but am just noticing this one now. It throws an exception so is it a debug level? I'm not sure what would happen in practice here - if it causes a job to completely quit then I'd say it would be error level, although I assume the exception would get logged somewhere anyway.

Reply:
I hink that is ok IMO the exeception may get logged but this is just doubley sure